README.rst

```diff
@@ -36,7 +36,7 @@ Documentation
 ~~~~~~~~~~~~~
 
 Documentation for ONGRConnectionsBundle can be found in 
-`here <http://ongr.readthedocs.org/en/latest/sources/ConnectionsBundle.git/Resources/doc/index.html>`_.
+`here <http://ongr.readthedocs.org/en/latest/>`_.
```

Comment:
Let's not use "here" here. It is not descriptive enough, so I would suggest using "ONGR handbook" or something like that.